pkg/models/responses/keydata.go

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright (C) 2024-2025 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/models"
+)
+
+// KeyDataResponse defines the Response Content for GET KeyData DTOs.
+type KeyDataResponse struct {
```

Comment:
move this struct into models/keydata.go
we don't have to create a new response package for this

Reply:
modified